core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/PeriodicDeltaChangeListener.java

```diff
@@ -382,6 +401,8 @@ protected void persistNowInternal(boolean alreadyHasMutex) {
             if (!alreadyHasMutex) persistingMutex.acquire();
             if (!isActive() && state != ListenerState.STOPPING) return;
             
+            updatePlaneIdIfTimedOut();
```

Comment:
Hm; if they are lossy then for planeId would we need to write it using a "keep trying and double-check until successful" approach?  I.e. don't progress until write is confirmed?